src/audio/module_adapter/module_adapter.c

```diff
@@ -1088,9 +1088,6 @@ static int module_adapter_copy_dp_queues(struct comp_dev *dev)
 		struct sof_source *data_src = dp_queue_get_source(dp_queue);
 		size_t dp_data_available = source_get_data_available(data_src);
 
-		if (!dp_data_available)
```

Comment:
this error is a real error - if DP processing is working it will never appear, even during pipelie startup